src/sql/workbench/contrib/notebook/browser/notebookActions.ts

```diff
@@ -310,9 +310,22 @@ export class RunParametersAction extends TooltipFromLabelAction {
 		let defaultParameters = new Map<string, string>();
 		editor.cells.forEach(cell => {
 			if (cell.isParameter) {
+				if (!cell.source) {
```

Comment:
Source can be an array of strings too - this doesn't seem to be handling the case where that's empty

Reply:
Should be covered by checking the regex test for each line in the parameter cell.